tests/network/vmi_slirp_interface.go

```diff
@@ -49,9 +49,10 @@ var _ = SIGDescribe("[Serial]Slirp Networking", func() {
 	var virtClient kubecli.KubevirtClient
 	var currentConfiguration v1.KubeVirtConfiguration
 
-	var genericVmi *v1.VirtualMachineInstance
-	var deadbeafVmi *v1.VirtualMachineInstance
+	var vmi *v1.VirtualMachineInstance
```

Comment:
You should be able to remove the `Serial` tag now from these tests to allow parallel execution.

Reply:
Fixed.